CONTRIBUTING.md

```diff
@@ -49,7 +49,7 @@ Here is what a `changeset.json` looks like:
     { "name": "@keystone-alpha/adapter-mongoose", "type": "patch" },
     { "name": "@keystone-alpha/keystone", "type": "minor" }
   ],
-  "dependents": []
+  "dependants": []
```

Comment:
`dependents` is the spelling that's used by changesets

Reply:
Good pickup - I had a code spell checker and overlooked this